node_modules/gh-rest/lib/rest/admin.js

```diff
@@ -81,6 +81,22 @@ module.exports = function(client) {
     };
 
     /**
+     * Change a global administrator's password
+     *
+     * @param  {Number}             id                          The id of the global administrator for which to change the password
+     * @param  {String}             newPassword                 The new password for the global administrator
+     * @param  {Function}           callback                    Standard callback function
+     * @param  {Object}             callback.err                An error that occurred, if any
+     */
+    client.admin.changePassword = function(id, newPassword, callback) {
+        var url = '/api/admins/' + client.util.encodeURIComponent(id) + '/password';
```

Comment:
It would be nice to have something like a `uprintf` (the `u` is for "url") that is similar to `sprintf`/`util.format`, except it will run `encodeURIComponent` on all the parameters:

`var url = uprintf('/api/admins/%s/password', id);`